packages/relay/src/lib/db/repositories/hbarLimiter/ipAddressHbarSpendingPlanRepository.ts

```diff
@@ -102,21 +102,6 @@ export class IPAddressHbarSpendingPlanRepository {
     }
   }
 
-  /**
```

Comment:
Nice could you also please remove the deleteAll() in the `EthAddressHbarSpendingPlanRepository` class also? 
It was added as an attempt to synchronize the two classes.

Reply:
resolved